packages/1155-contracts/src/delegation/ZoraCreator1155Attribution.sol

```diff
@@ -95,6 +95,12 @@ library ZoraCreator1155Attribution {
     string internal constant VERSION_2 = "2";
     bytes32 internal constant HASHED_VERSION_2 = keccak256(bytes(VERSION_2));
 
+    function allVersions() internal pure returns (string[] memory versions) {
+        versions = new string[](2);
```

Comment:
does this need to be in a separate function from `supportedPremintSignatureVersions()`?

Reply:
good q - its called via an external lib call, which saves codespace.

Reply:
ive actually got rid of this method and extra level of indirection and just flattened it into:

`DelegatedTokenCreation.supportedPremintSignatureVersions`